interface/forms/eye_mag/help.php

```diff
@@ -33,8 +33,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="description" content="Eye Exam Help" />
     <meta name="author" content="openEMR: ophthalmology help" />
-    <?php Header::setupHeader(); ?>
```

Comment:
hi @stu01509 , This looks odd. Why are we removing this here?

Reply:
Thanks for your mention, It seems like when I deal with conflict removed the wrong code. I have updated the commit, please review it again.